test/e2e/e2e_test.go

```diff
@@ -91,13 +93,32 @@ var _ = BeforeSuite(func() {
 	deprovision = ctx.MustProvision(ctx.Ctx())
 	ctx.MustInstall(ctx.Ctx())
 
+	// Create the test namespace if it doesn't exist
+	Eventually(func() error {
+		err := ctx.Ctx().Client().Create(context.Background(), &corev1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: testNamespace,
+				Annotations: map[string]string{
+					"olm.e2e": "no-e2e-kube-client-reset",
+				},
+			},
+		})
+		if err != nil && !k8serrors.IsAlreadyExists(err) {
+			return err
+		}
+		return nil
+	}).Should(Succeed())
+
 	var groups operatorsv1.OperatorGroupList
 	Expect(ctx.Ctx().Client().List(context.Background(), &groups, client.InNamespace(testNamespace))).To(Succeed())
 	if len(groups.Items) == 0 {
 		og := operatorsv1.OperatorGroup{
 			ObjectMeta: metav1.ObjectMeta{
-				Name:      "opgroup",
+				Name:      "global-operators",
```

Comment:
Some of the tests expect the default `operatorGroup` in the `testNamespace`, so I've changed the name here. Alternatively, I could write the tests to use any `operatorGroup` in the namespace, but there are some assumptions about the default scope of that `operatorGroup`.